starknet_py/tests/e2e/account/account_test.py

```diff
@@ -123,6 +123,8 @@ async def test_account_estimate_fee_for_declare_transaction(
     )
 
 
+# TODO (#1419): Fix contract redeclaration
+@pytest.mark.skip(reason="Redeclaration occurred")
```

Comment:
We can modify this test to estimate fee for a different transaction than declare and that will be sufficient.